### PR TITLE
ILS / LOC Fixes

### LINKED ISSUE #2198: KCRG
# Airport ICAO
KCRG
# Runway(s) with ILS
32
# References (Official sources/charts)
https://aeronav.faa.gov/d-tpp/2103/00208IL32.PDF
# Are you working on this?
No

### LINKED ISSUE #2212: DIAP
# Airport ICAO
DIAP

# Runway(s) with ILS
21
# References (Official sources/charts)
https://aim.asecna.aero/html/index-fr-FR.html 

# Are you working on this?

Yes

### LINKED ISSUE #2226: KMDT
# Airport ICAO
KMDT

# Runway(s) with ILS
13, 31
# References (Official sources/charts)
https://flightaware.com/resources/airport/KMDT/IAP/ILS+OR+LOC+RWY+31
https://flightaware.com/resources/airport/KMDT/IAP/ILS+OR+LOC+RWY+13

# Are you working on this?

Yes

### LINKED ISSUE #2227: KMAF
# Airport ICAO
KMAF

# Runway(s) with ILS
10
# References (Official sources/charts)
https://flightaware.com/resources/airport/KMAF/IAP/ILS+OR+LOC+RWY+10

# Are you working on this?

Yes

### LINKED ISSUE #2228: KPMD
# Airport ICAO
KPMD

# Runway(s) with ILS
25
# References (Official sources/charts)
https://flightaware.com/resources/airport/KPMD/IAP/ILS+OR+LOC+RWY+25

# Are you working on this?

Yes

### PR DESCRIPTION
**Pull request by suhasjatla**
KCRG - (Fixed #2198)
DIAP - (Fixed #2212)
KMDT - (Fixed #2226)
KMAF - (Fixed #2227)
KPMD - (Fixed #2228)

### CODE REVIEW

**Review by MajorTomMueller — APPROVED** [review]

File: Glideslope.json [hunk 1]
```diff
@@ -1866,10 +1866,10 @@
    "associatedRunwayNumber": "21",
    "frequency": 110300,
    "elevation": 20,
-   "latitude": 5.2737943,
-   "longitude": -3.92069811,
-   "bearing": 202.32,
-   "glideslope": 3.0020232,
+   "latitude": 5.27378051,
+   "longitude": -3.92070316,
+   "bearing": 22.34,
```
Comment (MajorTomMueller) [inline Glideslope.json:1871]: This is wrong, runway number 21 must have a bearing of 205 to 215

File: LOC.json [hunk 2]
```diff
@@ -1865,9 +1865,9 @@
    "associatedRunwayNumber": "21",
    "frequency": 110300,
    "elevation": 20,
-   "latitude": 5.2737943,
-   "longitude": -3.92069811,
-   "bearing": 202.32,
+   "latitude": 5.27378051,
+   "longitude": -3.92070316,
+   "bearing": 22.34,
```
Comment (MajorTomMueller) [inline LOC.json:1870]: Same as in glideslope either the heading or the runway number is wrong

**Review by MajorTomMueller — CHANGES_REQUESTED** [review]
Beside bearings check also the identifiers.

File: LOC.json (hunk 2 above)
Comment (suhasjatla) [reply LOC.json:1870]: Bearing fixed; my mistake on that one.

**Comment by suhasjatla** [comment]
> Beside bearings check also the identifiers.

All charts I can find are listing the identifier as 'AN', so it should be correct.

**Review by MajorTomMueller — APPROVED** [review]

**Pull request merged**